src/devices/GrovePi/GrovePi.cs

```diff
@@ -0,0 +1,190 @@
+﻿using Iot.Device.GrovePiDevice.Models;
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.IO;
+using System.Threading;
+
+namespace Iot.Device.GrovePiDevice
+{
+    /// <summary>
+    /// Create a GrovePi class
+    /// </summary>
+    public class GrovePi : IDisposable
+    {        
+        private I2cDevice _i2cDevice;
+        private readonly bool _autoDispose;
+
+        /// <summary>
+        /// The default GrovePi I2C address is 0x04
+        /// Other addresses can be use, see GrovePi documentation
+        /// </summary>
+        public static byte GrovePiSefaultI2cAddress => 0x04;
+
+        /// <summary>
+        /// The maximum ADC, 12 bit so 2013 on GrovePi
```

Comment:
confused by the comment, since value is 1023

Reply:
typo, it's 1023

Reply:
isn't that 10bit though?

Reply:
yes, it is. copy/paste from GoPiGo with mistakes. correcting.